silta/node.Dockerfile

```diff
@@ -1,5 +1,14 @@
 FROM wunderio/silta-node:20-alpine-v1
 
-COPY . /app
+# Because we are using the standalone mode, we copy only the needed files
+# to the image.
+COPY ./.next/standalone /app
+COPY ./.next/static /app/.next/static
+COPY ./public /app/public
 
-CMD npm run start
+WORKDIR /app
+
+ENV PORT 3000
+
+# In standalone mode, we cannot use npm run start, we have to use the server.js file
```

Comment:
```suggestion
# server.js is created by next build from the standalone output
```

Reply:
Wait, I didn't know about this way of suggesting changes! (where have I been!) :D